README.md

```diff
@@ -1,8 +1,33 @@
+</div>
+
+<div align="center"> 
+ 
+	
+<a href="https://github.com/siv2r/kidney-exchange"><img src="https://img.shields.io/badge/Open%20Source-%F0%9F%A4%8D-Green"></a>
+<a href="https://github.com/siv2r/kidney-exchange"><img src="https://img.shields.io/badge/Built%20by-developers%20%3C%2F%3E-0059b3"></a>
+<a href="https://github.com/siv2r/kidney-exchange"><img src="https://img.shields.io/static/v1.svg?label=Contributions&message=Welcome&color=yellow"></a>
+<a href="https://github.com/siv2r/"><img src="https://img.shields.io/badge/maintenance-yes-brightgreen"></a>
+<a href="https://github.com/siv2r/kidney-exchange/blob/master/LICENSE"><img src="https://img.shields.io/github/license/siv2r/kidney-exchange?color=brightgreen"></a>
+	
+<a href="https://github.com/siv2r/kidney-exchange/graphs/contributors"><img src="https://img.shields.io/github/contributors/siv2r/kidney-exchange?color=brightgreen"></a>
+<a href="https://github.com/siv2r/kidney-exchange/stargazers"><img src="https://img.shields.io/github/stars/siv2r/kidney-exchange?color=0059b3"></a>
+<a href="https://github.com/siv2r/kidney-exchange/network/members"><img src="https://img.shields.io/github/forks/siv2r/kidney-exchange?color=yellow"></a>
+<a href="https://github.com/siv2r/kidney-exchange/issues"><img src="https://img.shields.io/github/issues/siv2r/kidney-exchange?color=0059b3"></a>
+<a href="https://github.com/siv2r/kidney-exchange/issues?q=is%3Aissue+is%3Aclosed"><img src="https://img.shields.io/github/issues-closed-raw/siv2r/kidney-exchange?color=yellow"></a>
+<a href="https://github.com/siv2r/kidney-exchange/pulls"><img src="https://img.shields.io/github/issues-pr/siv2r/kidney-exchange?color=brightgreen"></a>
+<a href="https://github.com/siv2r/kidney-exchange/pulls?q=is%3Apr+is%3Aclosed"><img src="https://img.shields.io/github/issues-pr-closed-raw/siv2r/kidney-exchange?color=0059b3"></a> 
+
+<a href="https://github.com/topics/html5"><img alt="HTML5" src="https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white"/></a>
+<a href="https://github.com/topics/php"><img alt="PHP" src="https://img.shields.io/badge/php-%23777BB4.svg?&style=for-the-badge&logo=php&logoColor=white"/></a> 
+<a href="https://github.com/topics/javascript"><img alt="JavaScript" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/></a> 
+<a href="https://github.com/topics/python"><img alt="Python" src="https://img.shields.io/badge/python%20-%2314354C.svg?&style=for-the-badge&logo=python&logoColor=white"/></a> 
+<a href="https://github.com/topics/css3"><img alt="CSS3" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white"/></a>
+<a href="https://github.com/topics/bootstrap"><img alt="BootStrap" src="https://img.shields.io/badge/Bootstrap-563D7C?style=for-the-badge&logo=bootstrap&logoColor=white"/></a> 
+</div>
```

Comment:
Can you move these new changes to the top of the file?

Reply:
@siv2r did u mean before the logo or at the top (before the heading)??

Reply:
@Umesh-01 before the heading (line 1 of this page)

Reply:
done, check it now @siv2r